README.md

```diff
@@ -138,6 +138,47 @@ Will result in:
 </div>
 ```
 
+## Custom widgets
+
+You can provide your own custom widgets to a uiSchema for the following json data types:
+
+- `string`
+- `number`
+- `integer`
+- `boolean`
+- `date-time`
+
+```js
+const schema = {
+  type: "string"
+};
+
+const uiSchema = {
+  widget: (props) => {
```

Comment:
Isn't it clearer if you read the whole section instead of just the diff as a normal user would do? https://github.com/mozilla-services/react-jsonschema-form/blob/custom-widgets/README.md#alternative-widgets